backend/generalhw.pm

```diff
@@ -24,7 +24,6 @@ use autodie ':all';
 
 use base 'backend::baseclass';
 
-use bmwqemu 'diag';
```

Comment:
removing this line affects the using `bmwqemu::diag()` in line 84?

Reply:
good catch, thx.

Reply:
fixed